index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Chef Hannah's Site</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body style ="background-color: lightskyblue;">
+
+   <!--navigation bar and color changing text-->
+    <div class="title-container">
+            <h1>welcome to hannah's recipe website! </h1>   
+            <nav>
+                <a class = "nav-item" href="./index.html">home</a>
+                <a class = "nav-item" href="./about.html">about me</a>
+            </nav>
+
+    </div>
+
+
+<!--banana bread intro card-->
+
+    <div class = "recipe-container">
+        <h2>banana bread</h2>
+        <div class = "flex-container">
+        <img
+      src="https://www.allrecipes.com/thmb/GBkEPttyEbnx2x8tcRzkabMGYi0=/2000x2000/filters:no_upscale()/20144-banana-banana-bread-mfs-60-bddcb8e0caac452386de52f6fecf33db.jpg"
+      alt="Banana Bread Image Here"
+      width="350"
+    />
+        </div>
+        <p>decadant sweet cake made with bananas. pairs great with a scoop of vanilla ice cream! </p>   
+        
+        <a href="/Users/hannahmoshtaghi/bootcamp-project-2022/bootcamp-project-2022/bananaBread.html">
```

Comment:
You should use a relative `href` here like you do for your navbar links! Otherwise, people won't be able to access the link because they're not hannahmoshtaghi!